velox/connectors/hive/TableHandle.h

```diff
@@ -178,6 +179,11 @@ class HiveTableHandle : public ConnectorTableHandle {
     return tableParameters_;
   }
 
+  // Full schema including partitioning columns and data columns.
```

Comment:
Does this need to be full schema or only columns that are used in subfieldFilters and remainingFilter?